### PR TITLE
Docs 2468 afm

### PR DESCRIPTION
**Pull request by rddheath**
This pull request is for the updates to the AFM documentation for the 7/12:7/14 release. 

This documentation will be published on 7/14.

### CODE REVIEW

File: api-functional-monitoring/v/latest/afm-create-monitor.adoc [hunk 1]
```diff
@@ -42,15 +42,15 @@ To create a monitor for a test of one endpoint:
 
 == Result
 
-The dashboard opens. The monitor is listed on the left. Beneath the name of the monitor is an indicator that explains when the current run of the monitor began. A line at the bottom of the list item shows how much time remains before the next run of the monitor. Also, the *Monitored Endpoints*, *Tests*, and *Schedules* sections will all be empty. After the monitor has run, the list item for the test suite will indicate whether the monitored endpoint passed or failed.
+The dashboard opens. The monitor is listed on the left. Beneath the name of the monitor is an indicator that explains when the current run of the monitor began. A line at the bottom of the list item shows how much time remains before the next run of the monitor. Also, the *Monitored Endpoints*, *Tests*, and *Schedules* sections will all be empty. After the monitor has run, the list item for the test suite will indicate whether the monitored endpoints passed or failed.
```
Comment (sleslie23) [inline api-functional-monitoring/v/latest/afm-create-monitor.adoc]: Minor comment: I found this a little hard to parse. Maybe a bulleted list?

**Comment by sleslie23** [comment]
LGTM!
